runtime/JavaScript/spec/rewriter/Makefile

```diff
@@ -0,0 +1,16 @@
+ANTLR_VERSION = 4.12.0
```

Comment:
I think we have a small problem here
The purpose if the CI is not to test against the latest official version, rather against the version being tested.
So the makefile should use the tool from the local maven repo, not from the antlr web site.

Reply:
OK, how do I do that? Are there any examples within the repo that I can follow? I can see that the Java tests are capable of generating lexers/parsers from a grammar at run-time but what about other languages?

When I started working on this, I found Python tests using generated code (runtime/Python2/tests/mocks/TestLexer.py) that has been commited to the repo, so I assumed this was OK.

Reply:
I think my contribution ends here. Over a year ago, I found myself needing a JavaScript version of the TokenStreamRewriter. That is no longer the case, so I have no vested interest in putting more effort into this.

Reply:
All generic grammar tests generate parsers at CI time, but they are run from java.

Every GitHub action runs the "Build ANTLR with Maven" step, which builds the tool and registers it in the local maven repo, which should be found at ~/m2/repository/org/antlr/antlr4/${version}/antlr4-${version}-complete.jar.

You are correct in observing that many target-specific tests do not follow the expected policy, but let's not take things worse :-)

Reply:
Sorry to hear that, and apologies for taking so long to consider this PR. I hope your interest will grow again soon. This is useful work for the community.